### PR TITLE
CC 11: Perlin Noise Terrain

### PR DESCRIPTION
**Pull request by dipamsen**
- [x] timestamps
- [x] showcase

image

image

### CODE REVIEW

**Comment by klinegareth** [comment]
Is this ready for review @dipamsen ?

**Comment by dipamsen** [comment]
Yes! @klinegareth

File: content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    { "time": "02:14", "title": "Code! Draw a rectangular grid!" },
+    { "time": "04:13", "title": "Code! Create a flat triangle strip mesh!" },
+    { "time": "06:51", "title": "Code! Rotate the surface in 3D!" },
+    { "time": "08:47", "title": "Code! Set the z-values of the vertices randomly!" },
+    { "time": "11:04", "title": "Code! Create a 2D array to store the z values!" },
+    { "time": "13:31", "title": "Explain! How do we make the terrain infinite? What is Perlin Noise?" },
+    { "time": "17:03", "title": "Explain! How do we make the terrain smooth?" },
+    { "time": "17:43", "title": "Code! Reduce offsets to get smoother z values" },
+    { "time": "19:17", "title": "Explain! How do we make it appear as if we are moving over the terrain?" },
+    { "time": "20:30", "title": "Code! Change y-offset per frame make the illusion of flying!" },
```
Comment (klinegareth) [inline content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json:23]:
```suggestion
    { "time": "20:30", "title": "Code! Change y-offset per frame to make the illusion of flying!" },
```

File: content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  "description": "In this coding challenge, I create a 3D procedural terrain using Perlin Noise and the beginShape() function in Processing.",
+  "videoNumber": "11",
+  "videoId": "IKB1hWWedMk",
+  "date": "2016-05-04",
+  "languages": ["Processing"],
+  "topics": ["perlin noise", "terrain generation", "procedural"],
+  "canContribute": true,
+  "relatedChallenges": [],
+  "timestamps": [
+    { "time": "00:00", "title": "Welcome! Introduction to the Challenge" },
```
Comment (klinegareth) [inline content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json:12]: I think for timestamps on the site we're just doing `Introduction to the Challenge`, `What do we need to do?` etc. without the `Welcome!` or `Explain!` part

Comment (dipamsen) [reply content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json]: Thanks for the suggestions, will keep this in mind for the future.

Comment (shiffman) [reply content/videos/challenges/11-3d-terrain-generation-with-perlin-noise/index.json]: Thanks for raising this @klinegareth! I've gone through a variety of different styles fo the timestamps. I think I've settled on preferring timestamps that are concise and to the point so even just "Introduction" would work for `00:00`. Open to your ideas and no need to make these changes now!

**Pull request merged**